packages/PoppyPrint-Core.package/PPFormatter.class/instance/braceNodeHasMultiLineParts..st

```diff
@@ -0,0 +1,5 @@
+helper
+braceNodeHasMultiLineParts: aNode
+
+	^ (aNode elements anySatisfy: [:node |
+		(self willBeMultiLine: node) or: (self isTypeOfNewLineMarker: node)]) or: [self isCaseOf: (self parentFor: aNode)]
```

Comment:
The convention for for writing logical statements including `and:` and `or:` ist the following:

- `statement1 or: [statement2]` not `statement1 or: statement2`
- `statement1 and: [statement2]` not `statement1 and: statement2`

This is because when evaluating `statement1 or: statement2` the `statement2` is always evaluated first bevor `statement1 or: statement2` can be evaluated, because 'or:' needs the value of its Argument.
If some would instead evaluate `statement1 or: [statement2]` then `statement2` does not have to be evaluated because it is contained in a Block. So in this case `statement1` is evaluated first and only if `statement1` is `flase` then `statement2` will be evaluated.